doc/nvim-tree-lua.txt

```diff
@@ -43,6 +43,7 @@ CONTENTS                                                           *nvim-tree*
   6.7  API Marks                                |nvim-tree-api.marks|
   6.8  API Config                               |nvim-tree-api.config|
   6.9  API Commands                             |nvim-tree-api.commands|
+  6.10 API Log                                  |nvim-tree-api.log|
```

Comment:
```suggestion

```
There is no need for that. No other commands have separate entry in TOC, nor should this one.